src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -113,6 +114,15 @@ public void disconnected(ExecutorDriver driver) {
 
     @Override
     public void launchTask(final ExecutorDriver driver, final TaskInfo task) {
+        // "task id beginning with "yarn" is a mesos task for yarn container
+        if (task.getTaskId().getValue().startsWith("yarn")) {
```

Comment:
The opposite actually, mesos will resize the container.  I spent some time finding the info so the comment was ment to save others the trouble.